heroic-core/src/main/java/com/spotify/heroic/metric/LocalMetricManager.java

```diff
@@ -179,17 +179,24 @@ public boolean isEmpty() {
             final DataInMemoryReporter dataInMemoryReporter = reporter.newDataInMemoryReporter();
 
             final QuotaWatcher watcher = new QuotaWatcher(
-                options.getDataLimit().orElse(dataLimit).asLong().orElse(Long.MAX_VALUE), options
-                .getAggregationLimit()
-                .orElse(aggregationLimit)
-                .asLong()
-                .orElse(Long.MAX_VALUE), dataInMemoryReporter);
+                options.getDataLimit()
+                       .orElse(dataLimit)
+                       .asLong()
+                       .orElse(Long.MAX_VALUE),
+                options.getAggregationLimit()
+                       .orElse(aggregationLimit)
+                       .asLong()
+                       .orElse(Long.MAX_VALUE),
+                dataInMemoryReporter
+            );
```

Comment:
This is a perfect example of what we're discussing in the channel right now. We choose to obey a tool and some rules rather than aim for readability.